### ISSUE TITLE
CMD for centos-zeppelin should be 'upstart'?

### ISSUE THREAD

**Issue by cupdike** [internavenue/dockerfiles#2]
For the centos-zeppelin Dockerfile, should it be:
`CMD ["bin/zeppelin-daemon.sh", "upstart"]`

instead of;
`CMD ["bin/zeppelin-daemon.sh", "start"]`
?

With start, the command returns and the container stops.  With upstart, it starts Zeppelin in the foreground and continues running (what I would expect).  Or am I using it incorrectly?

**Comment by jmrr** [comment]
You're right, there was a [JIRA issue](https://issues.apache.org/jira/browse/ZEPPELIN-641) created after I wrote the Dockerfile, which now reflects the update.

Note however that I will modify the Dockerfile so the Zeppelin volume is mapped to a volume on the host so the data becomes indpendent of the running container. This suits better our needs but makes this image less standalone.
